cloverage/src/cloverage/sample/exercise_instrumentation.clj

```diff
@@ -5,8 +5,12 @@
 
 '()
 
+(+ 40)
+
 (+ 40 2)
 
+(str 1 2 3)
+
```

Comment:
Because `(+ 40 2)` actually gets replaced with an inlined function call by the compiler I added some more forms that don't get replaced so we can test for those as well.